tests/Functional/Schema/SchemaManagerFunctionalTestCase.php

```diff
@@ -1418,6 +1419,58 @@ public function testCommentInTable(): void
         $table = $this->schemaManager->listTableDetails('table_with_comment');
         self::assertSame('Foo with control characters \'\\', $table->getComment());
     }
+
+    public function testCreatedCompositeForeignKeyOrderIsCorrectAfterCreation(): void
+    {
+        $localColumns = ['child_col2', 'child_col1'];
+        $foreignColumns = ['col2', 'col1'];
+
+        $table = new Table('test_table_foreign_key_referenced');
+
+        $table->addOption('collate', 'utf8_general_ci');
+        $table->addOption('charset', 'utf8');
+        $table->addOption('comment', 'Test foreign key referenced table');
```

Comment:
All the options above are irrelevant to the test. Please remove them.

Reply:
Are removed now.